frontend/src/components/object/ObjectPermission.vue

```diff
@@ -50,14 +50,21 @@ const updateObjectPermission = (value: boolean, userId: string, permCode: string
   else {
     const managers = getMappedObjectToUserPermissions.value.filter( (x: UserPermissions) => x.manage );
 
+    const userPerms: UserPermissions = getMappedObjectToUserPermissions.value
+      .find( (x: UserPermissions) => x.userId === userId ) as UserPermissions;
+
+    // Due to 2-way binding we check if there are no managers left when MANAGE is unticked
+    const noManagers = permCode === Permissions.MANAGE && !managers.length;
+
+    // When READ is unticked check if they are the last remaining user with MANAGE
+    const lastManager = userPerms.manage && permCode === Permissions.READ && managers.length === 1;
```

Comment:
Could also make this line `const lastManager = managers.length === 1 && managers[0].userId === userId;` I guess